sympy/geometry/point.py

```diff
@@ -442,32 +557,44 @@ def __add__(self, other):
 
         """
 
-        if iterable(other) and len(other) == len(self):
-            return Point([simplify(a + b) for a, b in zip(self, other)])
-        else:
-            raise ValueError(
-                "Points must have the same number of dimensions")
+        if iterable(other):
+            coords = [simplify(a + b) for a, b in zip(self, other)]
+            if len(coords) != self.ambient_dimension:
+                raise ValueError("Points must have the same number of dimensions")
+            if all(isinstance(c, Number) for c in coords):
```

Comment:
OK, I'm in agreement. When performing a calculation like `S(1)/3.0` one gets a Float so when multiplying a Point by a float I guess one shouldn't be surprised that it makes the Point inexact.